src/bosh-director-core/spec/bosh/director/core/templates/job_template_loader_spec.rb

```diff
@@ -103,19 +107,18 @@ module Bosh::Director::Core::Templates
           'monit',
           'monit',
           'monit file erb contents',
-          'plan-job-name',
+          'plop-job-name',
         ).and_return(monit_erb)
 
         expect(SourceErb).to receive(:new).with(
           'test',
           'test_dst',
           'test contents',
-          'plan-job-name'
+          'plop-job-name'
```

Comment:
Is there a reason for the change from `plan` => `plop`? If this is a meaningless identifier maybe use something like `FAKE_XYZ...` or `FAKE_123...`

Reply:
I’ll use `fake-job-name-1` and `fake-job-name-2` instead of plip/plop.